File: refacciones-frontend/src/types/refaccion.ts
export interface Refaccion {
    id: number;
    nombre: string;
    descripcion: string;
    precio: number;
    stock: number;
    categoria: string;
}

export interface CreateRefaccionDto {
    nombre: string;
    descripcion: string;
    precio: number;
    stock: number;
    categoria: string;
}

File: Refacciones.Domain/Entities/Refaccion.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Refacciones.Domain.Entities
{
    public class Refaccion
    {
        public int Id { get; set; }

        public string Nombre { get; set; } = string.Empty;

        public string Descripcion { get; set; } = string.Empty;

        public decimal Precio { get; set; }

        public int Stock { get; set; }

        public string Categoria { get; set; } = string.Empty;

        public DateTime FechaCreacion { get; set; } = DateTime.UtcNow;
    }
}
